src/configs/envs.ts

```diff
@@ -0,0 +1,67 @@
+import dotenv from 'dotenv';
+import path from 'path';
+
+// initialize configuration
+dotenv.config();
+
+export const EXPRESS_OPENSRP_ACCESS_TOKEN_URL =
```

Comment:
can you add a todo comment about this being specific to OpenSRP now, but explaining what you explained here so someone who reads this understand the future direction, https://onaio.slack.com/archives/CBSU8L2SC/p1586184103075900?thread_ts=1586183662.075400&cid=CBSU8L2SC